src/board.c

```diff
@@ -640,6 +550,9 @@ void Board_HandleLtc6804Status(LTC6804_STATUS_T status) {
             break;
         default:
             Board_Println("Entered default case in Board_HandleLtc6804Status(). You should never reach here");
+#ifdef FSAE_DRIVERS
+            Error_Assert(ERROR_CONTROL_FLOW, msTicks);
```

Comment:
This is because the ERROR_CONTROL_FLOW enum value is also ifdef-guarded. Can change this if the hooligans want the value as well.